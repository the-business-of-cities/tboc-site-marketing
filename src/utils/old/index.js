export * from "./bp";
export * from "./colors";
export * from "./shadow";
export * from "./sizing";
export * from "./styles";
export * from "./utility";