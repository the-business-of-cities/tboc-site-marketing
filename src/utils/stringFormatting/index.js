export const formatTelNumber = (num) => {
  if (num.slice(0, 1) === "+" && num.length === 13) {
    return `${num.slice(0, 3)} (0)${num.slice(3, 6)} ${num.slice(
      6,
      9
    )} ${num.slice(9, 13)}`;
  }

  if (num.slice(0, 1) === "0" && num.length === 11) {
    return `${num.slice(0, 4)} ${num.slice(4, 7)} ${num.slice(7, 11)}`;
  } else {
    return num;
  }
};
