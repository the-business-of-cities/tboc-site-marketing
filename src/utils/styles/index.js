export { default as defaultGlobalStyles } from "./defaultGlobalStyles";
export { default as theme } from "./themes/theme";
export * from "./breakpoints";
