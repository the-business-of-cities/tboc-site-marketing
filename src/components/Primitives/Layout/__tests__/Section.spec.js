import { Section } from "../index";
import React from "react";
import { mount } from '../../../../utils/tests';

describe("Section", () => {
  it("renders correctly", () => {
    expect(mount(<Section />)).toMatchSnapshot();
  });
});
