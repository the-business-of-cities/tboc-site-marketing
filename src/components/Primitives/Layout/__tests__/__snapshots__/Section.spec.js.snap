// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section renders correctly 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #fff;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-basis: 100%;
  -ms-flex-preferred-size: 100%;
  flex-basis: 100%;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
  background-size: cover;
}

.c0:nth-child(2n) {
  background-color: #fafafa;
}

@media (max-width:999px) {
  .c0 {
    -webkit-flex-wrap: wrap;
    -ms-flex-wrap: wrap;
    flex-wrap: wrap;
  }
}

<styled.div>
  <div
    className="c0"
  />
</styled.div>
`;
